spec/controllers/questionnaires_controller_spec.rb

```diff
@@ -1,4 +1,20 @@
-describe QuestionnairesController do
+extend RSpec::Matchers
```

Comment:
You do not need to add the code at the top of this test file. Please remove them.
`should` is a deprecated RSpec keyword, you need to use `expect` instead.